.github/workflows/CI2.yml

```diff
@@ -203,13 +221,15 @@ jobs:
       - name: Build kokkos
         working-directory: kokkos
         run: |
+          [[ ${{ matrix.heffte }} == *"ONEAPI"* ]] && kokkos_cmake_opts=( "-DCMAKE_CXX_STANDARD=17 -DKokkos_ENABLE_SYCL=ON" )
```

Comment:
The test gives an error on missing `libOpenCL.so`, which is part of OneAPI and should be installed.

Try sourcing the Intel setup script:
```
source /opt/intel/oneapi/setvars.sh
```
If things are still not working, I would try using `dpcpp` as the compiler in place of `icpc`